.github/workflows/deploy-phase1.yml

```diff
@@ -1,19 +1,19 @@
-name: STAGING - Deploy Phase1 API
+name: PROD - Deploy Phase 1 API
 on:
   push:
     branches:
       - phase1
 
 jobs:
   deploy:
-    name: Deploy Phase1 Staging API
+    name: Deploy Phase 1 API
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out Git repository
         uses: actions/checkout@v2
       - uses: akhileshns/heroku-deploy@v3.12.12
         with:
-          heroku_app_name: ${{ secrets.HEROKU_APP_NAME_STAGING_PHASE1 }}
```

Comment:
Looking into documentation, makes no mention that app name should be secret and these secrets seem to provide minimal protection especially for archive apps https://github.com/marketplace/actions/deploy-to-heroku